swagger-spec/draft_registrations/draft_registration_definition.yaml

```diff
@@ -11,22 +11,46 @@ properties:
     type: string
     readOnly: true
     description: 'The unique identifier of the draft registration entity.'
-
   type:
     type: string
     readOnly: true
     description: 'The type identifier of the draft registration entity (`draft_registrations`).'
-
   attributes:
     type: object
     title: Attributes
     readOnly: false
-    description: 'The properties of the draft registration entity.'
-    required:
-      - datetime_initiated
-      - datetime_updated
-      - registration_supplement
+    description: 'The properties of the Draft Registration entity.'
     properties:
+      title:
+        type: string
+        readOnly: false
+        description: 'The title of the Draft Registration.'
+      description:
+        type: string
+        readOnly: false
+        description: 'The description of the Draft Registration.'
+      category:
+        type: string
+        readOnly: false
+        description: 'The category of the Draft Registration.'
+      tags:
+        type: string
+        readOnly: false
+        description: 'The searchable tags of the Draft Registration.'
+      node_license:
+        type: string
```

Comment:
`node_license` is a nested object, not a string (the definitions for `nodes` and `registrations` also say it's a string, but with the [actual shape also described](https://github.com/CenterForOpenScience/developer.osf.io/blob/642d6a3d0a887568b299876d53f5f00945e1d906/swagger-spec/nodes/definition.yaml#L85)